FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/NDayMultiComputable.java

```diff
@@ -34,8 +34,8 @@ public float[] multiCompute(float[] values) {
         values = stepOneCompute.multiCompute(values);
 
         for(float day : days) {
-            int timeStep = (int) config.getDuration().getSeconds();
-            int timeStepDay = 86400 / timeStep;
+            int timeStepSeconds = (int) config.getDuration().getSeconds();
+            int timeStepDay = 86400 / timeStepSeconds;
             float interpVal = timeStepDay * day;
             results[i] = interpolateNDay(values, interpVal);
```

Comment:
Would the zero index corection make more sense here, since this is where we are calculaing the index to look for a value on?
```
float interpVal = timeStepDay * day-1;
```
```suggestion
            float interpVal = timeStepDay * day;
```

Reply:
But then `startIndex` and `endIndex` are not really the true indicies bounding the interpolation.

Reply:
Suggestion to have javadoc comment on the variable called numberDays[]   The names numberDays, and days, really feels like an integer, but defined as float.

Reply:
Added javadocs